### DISCUSSION TITLE
[Resolved] Vercel Deployment Issue: Continuous Loading While Integrating with Supabase and Empty Deploy Section

### DISCUSSION THREAD

**Discussion by yachty66** [vercel/community#5589] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/templates/next.js/headshot-ai

### Steps to Reproduce

I am trying to deploy a [[template](https://vercel.com/templates/next.js/headshot-ai)](https://vercel.com/templates/next.js/headshot-ai). However, when I try to add an integration to Supabase, it does not finish and keeps loading. It appears like ![this](https://github.com/vercel/community/assets/68147541/877bfe94-757f-4f51-b7fc-fd6fe1c13fa5). Additionally, the deploy section is also empty, looking like ![this](https://github.com/vercel/community/assets/68147541/62265ddb-0b16-4f43-bcc4-a53218143fab). Why is it not working? Does Vercel have an issue right now?

**Comment by LeoLin4258** [comment]
same here

**Comment by yachty66** [comment]
any updates?

**Comment by kamilogorek** [comment]
The issue has been fixed and everything should be operational.

**Comment by nozma-knows** [comment]
Just tried using the Supabase integration and I'm still running into an error:

`Failed to sync environment variables: {"error":{"code":"not_found","message":"Project not found."}}`

Has anyone else been encountering this?

**Reply by nozma-knows** [reply]
![image](https://github.com/vercel/community/assets/58397985/dde69e93-41cf-4e58-ad86-1c355a962156)

**Comment by yachty66** [comment]
solution for me was to have maximum one active database in supabase

**Comment by patrickdundas** [comment]
@amyegan why was this closed, it seems to be impacting all vercel -> supabase template deploys for me in both Chrome and Firefox

**Comment by armenavanesi** [comment]
Same issue. Why was this closed?

**Comment by amyegan** [comment]
This discussion was closed when the original problem reported here was resolved months ago. It seems there may be a new problem causing similar behavior. 

Vercel does not own the integration code. If you're currently seeing problems with the Supabase integration, please reach out to them so they know what error you're seeing.
- https://supabase.com/support
- https://github.com/orgs/supabase/discussions
- https://discord.supabase.com

**Comment by patrickdundas** [comment]
I've reached out to supabase. Hoping we can get a fix on either end as this is breaking every deploy button on every vercel guide, example, and template that uses the supabase integration. 

Some prioritization or triage on vercel's side would likely be in their best interest too

https://github.com/supabase/supabase/discussions/20751#discussioncomment-9011130

**Comment by kamilogorek** [comment]
Hey everyone, can you retest your deployments and/or provide exact steps you're taking when you were able to make it break? I just retested it multiple times on few different orgs, and I was unable to make it fail. Thanks

**Comment by jonmeggo** [comment]
Choose Add New project in Vercel. Use Supabase Starter template. add Git Repo. Add Supabase integration -> Install Integration -> Create Project. 

Looking in the Supabase console, I can see that it successfully creates the Supabase project, but the Vercel deployment hangs with the message:

Failed to sync environment variables: {"error":{"code":"not_found","message":"Project not found."}}

**Comment by jonmeggo** [comment]
Sorry - forgot to mention - trying to use West EU (London) as region

**Comment by jonmeggo** [comment, marked as answer]
FIXED! My issue was that I already had the Supabase integration setup (running for other Vercel projects). I went to the general integration settings and set Supabase to 'All Projects' rather than specific ones and tried the whole New Project process again and it worked. Hope that helps.

**Reply by kamilogorek** [reply]
Thanks! I'll see if we can make it more obvious when others hit this case

[a comment by an automated account is not shown]
